packages/oas-normalize/test/index.test.ts

```diff
@@ -271,13 +332,22 @@ describe('#validate', () => {
     );
   });
 
+  /* eslint-disable vitest/no-conditional-expect */
   it('should error out, and show all errors, when a definition has lots of problems', async () => {
     const o = new OASNormalize(require.resolve('./__fixtures__/invalid/openapi-very-invalid.json'), {
       enablePaths: true,
     });
 
-    await expect(o.validate()).rejects.toMatchSnapshot();
+    try {
+      await o.validate();
+      assert.fail();
+    } catch (err) {
+      expect(err).toBeInstanceOf(SyntaxError);
+      expect(err.message).toMatchSnapshot();
+      expect(err.details).toMatchSnapshot();
```

Comment:
i think you should be able to do this instead:
```suggestion
      expect(err).toMatchSnapshot();
```

Reply:
That unfortunately only snapshots the error message.

![Screenshot 2024-12-20 at 9 49 57 AM](https://github.com/user-attachments/assets/a5037c9d-51bc-4226-8665-9f2bc24e0a6a)

Reply:
ah okay — this technically also handles the `expect(err).toBeInstanceOf(SyntaxError);` assertion also but yeah two steps forward two steps back here so fine with this as is